packages/ai/streams/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from '../core/index';
 export * from '../errors/index';
+export * from '../util/constants';
```

Comment:
i wasn't sure with stuff like this to mark as @internal, use a separate package or something else?

Reply:
we will merge ui-utils and ai in v5, so some of this may not be relevant - what constants need exporting?

Reply:
resolved by making ai/internal export